tests/util/golinter/README.md

```diff
@@ -0,0 +1,53 @@
+# golinter
+
+This golinter checks if our test files follow customerized lint rules.
```

Comment:
I suppose we want to circulate the proposal doc before committing these. Perhaps you can put a placeholder README for now.

Reply:
Yes, this README is a placeholder. Once it is completed, I will put the link to README into lint result, so that developers could refer to this README to fix lint errors.